.travis.yml

```diff
@@ -73,6 +73,11 @@ script:
       else
         ./node_modules/.bin/grunt jest:normal
       fi
+      echo 'Testing in server-render (HTML generation) mode...'
+      printf '\nmodule.exports.useCreateElement = false;\n' \
```

Comment:
I personally would have used `echo` for this too but meh.

Otherwise it seems good.